vendor/k8s.io/kubernetes/openshift-kube-apiserver/enablement/intialization.go

```diff
@@ -78,4 +80,7 @@ func ForceGlobalInitializationForOpenShift() {
 	// TODO, we should scrub these out
 	bootstrappolicy.ClusterRoles = bootstrappolicy.OpenshiftClusterRoles
 	bootstrappolicy.ClusterRoleBindings = bootstrappolicy.OpenshiftClusterRoleBindings
+
+	// we need to have the authorization chain place something before system:masters
+	server.SkipSystemMastersAuthorizer()
```

Comment:
// disable implicitly added system/master authz, and turn it into another authz mode "SystemMasters", to be added via authorization-mode